web/src/test/java/no/nav/modiapersonoversikt/service/saker/SakerServiceImplTest.kt

```diff
@@ -91,16 +92,16 @@ class SakerServiceImplTest {
         every { bidragSakControllerApi.find(any()) } returns listOf(BidragSakDto(roller = listOf(), saksnummer = "123", erParagraf19 = false))
         every { unleashService.isEnabled(any<Feature>()) } returns false
 
-        mockkStatic(SubjectHandler::class)
-        every { SubjectHandler.getSubject() } returns Optional.of(Subject("12345678910", IdentType.EksternBruker, SsoToken.oidcToken("token", HashMap<String, Any?>())))
+//        mockkStatic(SubjectHandler::class)
+//        every { SubjectHandler.getSubject() } returns Optional.of(Subject("12345678910", MeldingFraDTO.IdentType.EksternBruker, SsoToken.oidcToken("token", HashMap<String, Any?>())))
         every { sakApiGateway.opprettSak(any()) } returns SakDto(id = "123")
 
         MDC.put(MDCConstants.MDC_CALL_ID, "12345")
     }
 
     @AfterEach
     fun destroy() {
-        unmockkStatic(SubjectHandler::class)
+//        unmockkStatic(SubjectHandler::class)
```

Comment:
Hvorfor kommenterer vi ut disse i stedet for å slette?

Reply:
Gikk bare litt fort i svingene. Funksjonaliteten er erstattet av `AuthContextExtention` nedenfor, så sletter det som er utkommentert.